accord-core/build.gradle

```diff
@@ -29,11 +29,13 @@ java {
   withSourcesJar()
 }
 
+compileJava {
```

Comment:
can you move the 1.8 logic to buildSrc/src/main/groovy/accord.java-conventions.gradle